rgo/git.py

```diff
@@ -112,13 +112,15 @@ def rev_parse(self, ref, short=False):
                               stdout=subprocess.PIPE)
         return proc.stdout.rstrip()
 
-    def describe(self, pkg=None, spec_version=None):
+    def describe(self, pkg=None, spec_version=None, version_from="git"):
```

Comment:
Having the default here be different from the "global" default is not nice at all, but I understand why and at this stage I digress :grin: